docker/paseo-parachain-launch-config.yml

```diff
@@ -0,0 +1,47 @@
+# A parachain-launch (https://github.com/open-web3-stack/parachain-launch) config
+# used in dev env
+# contains 2 relaychain nodes + 1 parachain node
+#
+# Relaychain Configuration
+relaychain:
+  image: parity/polkadot:v1.1.0
+  # TODO: `rococo-local` is used here as `paseo` has only "raw" chain spec which can't be further
+  #       manipulated by parachain-launch
+  #       we might need to switch to other launch tools
```

Comment:
`parachain-launch` is changed. Do we still need this?
https://github.com/litentry/litentry-parachain/pull/3044

Reply:
I forgot to remove this 😅

Reply:
Yes please remove this file